src/htrun/host_tests_runner/host_test_default.py

```diff
@@ -441,7 +464,8 @@ def process_code_coverage(key, value, timestamp):
                         callbacks__exit = True
                         event_queue.put(("__exit_event_queue", 0, time()))
                     elif key == "__exit_event_queue":
-                        # This event is sent by the host test indicating no more events expected
+                        # This event is sent by the host test indicating no more events
+                        # expected
```

Comment:
Lots of comments you've wrapped could do with full stops in this file, given they attempt to be complete sentences

Reply:
I'm only wrapping the comments to placate flake8, not changing text. I'll be on forever if I start fixing up the comments too.